File: testprojects/csharp/benchmark/domain/Indirect/ViolatingFrom/CallStaticMethodIndirect_MethodStaticMethod.cs
namespace Domain.Indirect.ViolatingFrom
{
	public class CallStaticMethodIndirect_MethodStaticMethod
	{
        private Domain.Indirect.Intermediate.BackgroundService bckg;

		public void Case5()
        {
            bckg = new Domain.Indirect.Intermediate.BackgroundService();
            string s = Domain.Indirect.IndirectTo.ServiceOne.GetsName();
		}
	}
}

File: testresources/java/accuracy/src/domain/indirect/allowedfrom/CallStaticMethodIndirect_MethodStaticMethod.java
package domain.indirect.allowedfrom;

import domain.indirect.intermediate.BackgroundService;

public class CallStaticMethodIndirect_MethodStaticMethod {
	
	private BackgroundService b;
	
	public CallStaticMethodIndirect_MethodStaticMethod() {
	}

	// Indirect invocation of a static method
	@SuppressWarnings("static-access")
	public void case5(){
		String s = b.getServiceOne().getsName();
		System.out.println(s);
		
	}
}
